rust/core-lib/src/styles.rs

```diff
@@ -220,4 +221,11 @@ impl ThemeStyleMap {
         self.styles.push(style.clone());
         result
     }
+
+    /// Load all themes inside themes folder and append it to
+    /// the default themes set
+    pub fn load_theme_set(&mut self, themes_dir: &Path) {
```

Comment:
Nit: for new API I'm trying to use `pub(crate)` for fns that are only going to be used internally. (One advantage of this is that `rustc` can emit a warning if a fn marked `pub(crate)` is unused).